crates/sui-sdk/src/crypto.rs

```diff
@@ -236,17 +219,3 @@ impl InMemKeystore {
         Self { keys }
     }
 }
-
-impl AccountKeystore for Box<dyn AccountKeystore> {
```

Comment:
this is redundant and confusing to use. we instead use `enum_dispatch` to dispatch the `AccountKeystore` trait to its specific Keystore enum